Automattic/wp-calypso
# Improve tracking of Akismet ecommerce conversions

## Proposed Changes
- Correct the type of conversion event for Google Ads
- Add GTM conversion tracking
- Add GA4 conversion tracking
- Add UA conversion tracking

## Testing Instructions

* Use store sandbox on your sandbox
* Checkout this PR and go to `http://calypso.localhost:3000/checkout/akismet/ak_plus_yearly_1?flags=ad-tracking`
* If in GDPR/CCPA zone - be sure to accept cookie banner
* Enable debug logs for analytics `localStorage.setItem( 'debug', 'calypso:analytics:*' );`
* Refresh the page and purchase the product
* You should see events `recordOrderInAkismetGTM: Record Akismet GTM purchase` and `recordOrderInAkismetGA: Record Akismet Purchase` in console log, with appropriate properties assigned

## Pre-merge Checklist
